spring-cloud-dataflow-tasklauncher/spring-cloud-dataflow-tasklauncher-sink/pom.xml

```diff
@@ -0,0 +1,352 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.cloud</groupId>
+		<artifactId>spring-cloud-dataflow-tasklauncher</artifactId>
+		<version>2.9.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>spring-cloud-dataflow-tasklauncher-sink</artifactId>
+	<properties>
+		<revision>3.1.0-SNAPSHOT</revision>
+		<stream-apps-core.version>${revision}</stream-apps-core.version>
+		<java-functions.version>1.1.0-SNAPSHOT</java-functions.version>
+		<apps.base-image>springcloud/baseimage:1.0.0</apps.base-image>
```

Comment:
We'd build images via paketo just like servers/ctr. Image build is in gh actions, not in poms anymore(didn't check is poms would actually try to create images).

Reply:
There may be some compatibility issues with the app generator plugin which expects container image metadata for generating the apps pom. I'll have to investigate further